config/stage.json

```diff
@@ -23,6 +23,7 @@
 		"happychat": true,
 		"help": true,
 		"help/courses": true,
+		"help/directly": true,
```

Comment:
Great tip. I'll drop the flag altogether for now and add back into the appropriate env once it's ready to test more publicly.